lib/OrdinaryDiffEqNewmark/test/runtests.jl

```diff
@@ -0,0 +1,71 @@
+using OrdinaryDiffEqNewmark, Test, RecursiveArrayTools, DiffEqDevTools, Statistics
+
+# Newmark methods with harmonic oscillator
+@testset "Harmonic Oscillator" begin
+    u0 = fill(0.0, 2)
+    v0 = ones(2)
+    function f1_harmonic!(dv, v, u, p, t)
+        dv .= -u
+    end
+    function harmonic_jac(J, v, u, weights, p, t)
+        J[1,1] = weights[1] * (0.0) + weights[2] * (-1.0)
+        J[1,2] = weights[1] * (0.0) + weights[2] * ( 0.0)
+        J[2,2] = weights[1] * (0.0) + weights[2] * (-1.0)
+        J[2,1] = weights[1] * (0.0) + weights[2] * ( 0.0)
+    end
+    function f2_harmonic!(du, v, u, p, t)
+        du .= v
+    end
```

Comment:
What is the intended way to enforce this automatically? We do not have a `SecondOrderODEFunction` analogue to `DynamicalODEFunction`.